src/globus_sdk/services/auth/client/service_client.py

```diff
@@ -1803,3 +1804,31 @@ def delete_scope(self, scope_id: UUIDLike) -> GlobusHTTPResponse:
                     :ref: auth/reference/#delete_scope
         """
         return self.delete(f"/v2/api/scopes/{scope_id}")
+
+    def get_consents(self, identity_id: str) -> GetConsentsResponse:
```

Comment:
I think we should accept UUIDs as well, with our UUIDLike type.

Reply:
Sure I can make that change.